svn/common.py

```diff
@@ -49,10 +51,13 @@ def run_command(self, subcommand, args, success_code=0,
 
         _logger.debug("RUN: %s" % (cmd,))
 
+        environment_variables = copy.deepcopy(os.environ)
```

Comment:
``os.environ.copy()`` should be enough because keys and values are strings and there is no nesting, doesn't it?